<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="mainwidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget
{
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 rgba(29, 95, 181, 0.6), stop: 0.5 rgba(2, 32, 41, 0.1), stop: 1 rgba(29, 95, 181, 0.6));
	border-radius: 20px; /* Rounded corners */
	border: 1px solid rgba(0, 0, 0, .5);
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
        <widget class="QWidget" name="buttonwidgetl" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
    background: none;
	border: none;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
           <widget class="QPushButton" name="pushButtonL">
            <property name="styleSheet">
             <string notr="true">QWidget
{
    background: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="screenwidget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
	 background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 rgba(29, 95, 181, 0.95), stop: 0.85 rgba(2, 32, 41, 0.95), stop: 1 rgba(29, 95, 181, 0.6));
	border: 1px solid black;
	font-size: 24px;
	color: black;
	
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignmentFlag::AlignHCenter">
           <widget class="QLabel" name="label_1">
            <property name="styleSheet">
             <string notr="true">QWidget
{
    background: none;
	border: none;
	color: white;
}</string>
            </property>
            <property name="text">
             <string>Home Screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QWidget
{
    background: none;
	border: none
}</string>
            </property>
            <property name="text">
             <string>This will change</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget
{
   background: none;
   border: none
}</string>
            </property>
            <property name="text">
             <string>This will change</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">QWidget
{
	background: none;
	border: none
}</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
        <widget class="QWidget" name="buttonwidgetr" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
    background: none;
	border: none;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
           <widget class="QPushButton" name="pushButtonR">
            <property name="styleSheet">
             <string notr="true">QWidget
{
    background: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bottomwidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="bottomleftwidget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 rgba(29, 95, 181, 0.6), stop: 0.5 rgba(2, 32, 41, 0.1), stop: 1 rgba(29, 95, 181, 0.6));
	border-radius: 20px; /* Rounded corners */
	border: 1px solid rgba(0, 0, 0, .5);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
           <widget class="QWidget" name="widget_11" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget
{
	background: none;
	border: none;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
              <widget class="OptionLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">QLabel
 {
    background: rgba(255, 255, 255, 0.8);
    border-radius: 20px; /* Rounded corners */
    
	padding: 15px;
	padding-right: 100px;
	font-size: 32px;
 	text-align: center; /* Horizontal center */
	color: #000000;
    font-family: Roboto, sans-serif;
    font-size: 18pt;
	letter-spacing: 2px; /* Adds space between characters */
    font-weight: 500; 
}</string>
               </property>
               <property name="text">
                <string>Suggestions</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="OptionLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">QLabel
 {
    background: rgba(255, 255, 255, 0.8);
    border-radius: 20px; /* Rounded corners */
    
	padding: 15px;
	padding-right: 100px;
	font-size: 32px;
 	text-align: center; /* Horizontal center */
	color: #000000;
    font-family: Roboto, sans-serif;
    font-size: 18pt;
	letter-spacing: 2px; /* Adds space between characters */
    font-weight: 500; 
}</string>
               </property>
               <property name="text">
                <string>Scenario</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="OptionLabel" name="label_7">
               <property name="styleSheet">
                <string notr="true">QLabel
 {
    background: rgba(255, 255, 255, 0.8);
    border-radius: 20px; /* Rounded corners */
    
	padding: 15px;
	padding-right: 100px;
	font-size: 32px;
 	text-align: center; /* Horizontal center */
	color: #000000;
    font-family: Roboto, sans-serif;
    font-size: 18pt;
	letter-spacing: 2px; /* Adds space between characters */
    font-weight: 500; 
}</string>
               </property>
               <property name="text">
                <string>Charts</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="OptionLabel" name="label_8">
               <property name="styleSheet">
                <string notr="true">QLabel
 {
    background: rgba(255, 255, 255, 0.8);
    border-radius: 20px; /* Rounded corners */
    
	padding: 15px;
	padding-right: 100px;
	font-size: 32px;
 	text-align: center; /* Horizontal center */
	color: #000000;
    font-family: Roboto, sans-serif;
    font-size: 18pt;
	letter-spacing: 2px; /* Adds space between characters */
    font-weight: 500; 
}</string>
               </property>
               <property name="text">
                <string>Alerts</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bottomrightwidget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget
{
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 rgba(29, 95, 181, 0.6), stop: 0.5 rgba(2, 32, 41, 0.1), stop: 1 rgba(29, 95, 181, 0.6));
	border-radius: 20px; /* Rounded corners */
	border: 1px solid rgba(0, 0, 0, .5);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OptionLabel</class>
   <extends>QLabel</extends>
   <header location="global">optionlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
